charts/airbyte/values.yaml

```diff
@@ -0,0 +1,380 @@
+## @section Common Parameters
+
+## @param nameOverride String to partially override airbyte.fullname template with a string (will prepend the release name)
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override airbyte.fullname template with a string
+##
+fullnameOverride: ""
+
+## Pods Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+## @param serviceAccount.annotations [object] Annotations for service account. Evaluated as a template. Only used if `create` is `true`.
+## @param serviceAccount.create Specifies whether a ServiceAccount should be created
+## @param serviceAccount.name Name of the service account to use. If not set and create is true, a name is generated using the fullname template.
+##
+serviceAccount:
+  create: true
+  annotations: {}
+  name: airbyte-admin
+
+## @section Webapp Parameters
+
+webapp:
+  ## @param webapp.replicaCount Number of webapp replicas
+  replicaCount: 1
+
+  ## @param webapp.image.repository The repository to use for the airbyte webapp image.
+  ## @param webapp.image.pullPolicy the pull policy to use for the airbyte webapp image
+  ## @param webapp.image.tag The airbyte webapp image tag. Defaults to the chart's AppVersion
+  image:
+    repository: airbyte/webapp
+    pullPolicy: IfNotPresent
+    tag: 0.29.13-alpha
```

Comment:
We'll also need to add this file to the bumpversion config so we catch these image tags.